plugins/topology/package.json

```diff
@@ -35,7 +35,7 @@
     "@material-ui/lab": "^4.0.0-alpha.45",
     "@patternfly/patternfly": "4.224.2",
     "@patternfly/react-charts": "6.94.18",
-    "@patternfly/react-core": "4.276.6",
+    "@patternfly/react-core": "^4.276.8",
```

Comment:
@debsmita1 do we want to automatically update this package to future minor/patch versions? IMO that can be risky as sometimes it might include breaking changes. @invincibleJai wdyt?